backend/tests/ops_site/cans/test_models.py

```diff
@@ -89,3 +90,22 @@ def test_ContractLineItemFiscalYear_name():
     )
 
     assert contract_line_item_for_2022.name == contract_line_item_name
+
+
+@pytest.mark.django_db
+def test_FundingPartner_str():
+    fundingPartner = FundingPartner.objects.create(
+        name="FundingPartnerName", nickname="NotTheRealName"
+    )
+    assert "FundingPartnerName" == str(fundingPartner)
+
+
+@pytest.mark.django_db
+def test_Portfolio_str():
+    portfolio = Portfolio.objects.create(
+        name="PortfolioNameHere",
+        description="",
+        status="Not-Started",
+        current_fiscal_year_funding=0.0,
+    )
+    assert "PortfolioNameHere" == str(portfolio)
```

Comment:
I think you want this in a different directory, like the comment on the other test updates.  Should be under the `portfolios` folder.

Reply:
ditto on missing it - fixed